spec/schemas/_common.yaml

```diff
@@ -1767,22 +1767,22 @@ components:
       type: object
       properties:
         open_contexts:
-          description: The number of open search contexts.
+          description: The number of open search contexts. 
           type: number
         query_current:
-          description: The number of shard query operations that are currently running.
+          description: The number of shard query operations that are currently running. 
           type: number
         query_time:
-          description: The total amount of time for all shard query operations.
+          description: The total amount of time for all shard query operations. 
           $ref: '#/components/schemas/Duration'
         query_time_in_millis:
-          description: The total amount of time for all shard query operations, in milliseconds.
+          description: The total amount of time for all shard query operations, in milliseconds. 
```

Comment:
"The total amount of time taken to complete all shard query operations, in milliseconds"?